src/armsgs.py

```diff
@@ -209,10 +215,17 @@ def warn(self, msg):
             self._log.write(self.cleancolors(premsg+dbgmsg+msg)+"\n")
         return
 
-    def bug(self, msg):
+    def bug(self, msg, close_pdf=None):
         """
         Print a bug message
         """
+        # Close PDF?
```

Comment:
Good suggestion.

It is so.

> On Apr 19, 2016, at 9:47 AM, Ryan Cooke <notifications@github.com> wrote:
> 
> In src/armsgs.py <https://github.com/PYPIT/PYPIT/pull/43#discussion_r60267224>:
> 
> >          """
> >          Print a bug message
> >          """
> > +        # Close PDF?
> I don't mind this approach, but can't we just grab the logger from within bug()
> 
> This would save the import armsgs in pypit.py, and close_pdf could then become a boolean.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/PYPIT/PYPIT/pull/43/files/c6ea2400c520399c388744a9f162fb2f3c13b557#r60267224>
> !DSPAM:826,5716614d174072321113683!

----------------------------------------------
J. Xavier Prochaska
UCO/Lick Observatory
1156 High St.
UC Santa Cruz
Santa Cruz, CA 95064

http://www.ucolick.org/~xavier/
http://imps.ucolick.org/
http://www.qpqsurvey.org/
http://k1dm3.ucolick.org/
http://www.ucolick.org/~xavier/IDL/index.html
https://github.com/pyigm/pyigm
https://github.com/linetools/linetools

831-459-2135 (Direct)
831-459-2991 (UCO/Lick Main)
831-459-5244 (Fax)